### PR TITLE
(MODULES-2103) Adds RAL for volume group

### PR DESCRIPTION
**Pull request by petems**

### CODE REVIEW

File: spec/unit/puppet/provider/volume_group/lvm_spec.rb [hunk 1]
```diff
@@ -8,6 +8,23 @@
...
   end
 
+  vgs_output = <<-EOS
+  VG       #PV #LV #SN Attr   VSize  VFree
+  VolGroup   1   2   0 wz--n- 19.51g    0
+  EOS
+
+  describe 'self.instances' do
+    before :each do
+      @provider.class.stubs(:vgs).returns(lvs_output)
```
Comment (mhaskel) [inline spec/unit/puppet/provider/volume_group/lvm_spec.rb]: Looks like this should be `vgs_output`

**Comment by mhaskel** [comment]
Hey @petems , other than the test failure this looks good, just let me know when you have that fixed. Thanks!

File: spec/unit/puppet/provider/volume_group/lvm_spec.rb (hunk 1 above)
Comment (petems) [inline spec/unit/puppet/provider/volume_group/lvm_spec.rb]: Gah, too much copypasta

**Comment by petems** [comment]
@mhaskel Fixed! :smile:

**Comment by mhaskel** [comment]
:+1: thanks!

**Pull request merged**